src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.cs

```diff
@@ -86,20 +86,51 @@ protected override string GetFilterText(ISymbol symbol, string displayText, CSha
             CompletionOptions options,
             CancellationToken cancellationToken)
         {
-            if (context != null && options.TriggerInArgumentLists)
+            if (context is null)
+                return true;
```

Comment:
when is the context null?

Reply:
Here's one location I found
https://github.com/dotnet/roslyn/blob/a35b73595c954a0281fac24e0b6136eca572db7a/src/Features/Core/Portable/Completion/Providers/AbstractRecommendationServiceBasedCompletionProvider.cs#L214

Reply:
thanks!